generics/messageConstants/apiResponses.js

```diff
@@ -368,4 +368,33 @@ module.exports = {
   SOLUTIONS_LIST: 'Solution Fetched successfully',
   KEYS_INDEXED_SUCCESSFULL: 'Keys indexed successfully',
   KEYS_ALREADY_INDEXED_SUCCESSFULL: 'Keys already indexed',
+  COURSE_LINK_REQUIRED: 'Course link is required',
```

Comment:
@praveenKDass make sure for newly added responses we already had similar message in this file. Lets avoid duplication

Reply:
Sure @VISHNUDAS-tunerlabs